### ISSUE TITLE
Confusing description of Tracking queries

### ISSUE THREAD

**Issue by space-alien** [dotnet/EntityFramework.Docs#3813]
["Query results don't contain any entity, which is added to the context but not yet saved to the database."](https://docs.microsoft.com/en-us/ef/core/querying/tracking#:~:text=Query%20results%20don%27t%20contain%20any%20entity%2C%20which%20is%20added%20to%20the%20context%20but%20not%20yet%20saved%20to%20the%20database.)

This is confusingly written. Does this mean to convey that a query will never return any entity that exists in the context in the Added state?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f3ffc114-df51-0be7-8972-6fcead14e011
* Version Independent ID: 94c34571-7314-aab1-7898-93ac6d0f3430
* Content: [Tracking vs. No-Tracking Queries - EF Core](https://docs.microsoft.com/en-us/ef/core/querying/tracking)
* Content Source: [entity-framework/core/querying/tracking.md](https://github.com/dotnet/EntityFramework.Docs/blob/main/entity-framework/core/querying/tracking.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @smitpatel
* Microsoft Alias: **avickers**

**Comment by roji** [comment]
Yes - queries return entities from the database, so any entity instance not yet saved won't be returned.

**Comment by space-alien** [comment]
Thanks Shay. Might I suggest rephrasing the documentation along these lines:

> When the results are returned in a tracking query, EF Core will check if the entity is already in the context. If EF Core finds an existing entity, then the same instance is returned. EF Core won't overwrite current and original values of the entity's properties in the entry with the database values. If the entity isn't found in the context, then EF Core will create a new entity instance and attach it to the context. ***Queries only return entities that already exist the database. Any unsaved entities in the context will not be included in query results.***
